README.md

```diff
@@ -294,6 +287,36 @@ Figure 11. VMClarity Dashboard
 
 <img src="img/vmclarity-dashboard-data.png" alt="VMClarity Dashboard with Findings" width="90%" height="90%" title="VMClarity Dashboard with Findings" />
 
+# Roadmap
+VMClarity project roadmap is available [here](https://github.com/orgs/openclarity/projects/2/views/7).
+
+# VMClarity Project Goals
+
+- **Increase the adoption of VMClarity**: One of the primary goals of VMClarity
+  is to reduce the number of vulnerable VMs in the world. We hope to do this by
+  getting more people involved in the project. This involves feature development,
+  marketing efforts, improving the user experience, and providing additional
+  documentation and resources to make it easier for users to get started.
+
+- **Improve the accuracy of VMClarity**: Another goal is to enhance the
+  accuracy across all supported types of threat detection by improving the
+  analysis and result merging logic performed by VMClarity to reduce false
+  positives. This involves researching and developing improved algorithms and
+  heuristics used by VMClarity.
+
+- **Expand VMClarity's functionality**: A third goal is to expand the
+  functionality of VMClarity by adding additional tools to each family, adding
+  additional families of threats to detect, and integrating with other security
+  tools to provide a more comprehensive security solution.
+
+- **Foster a community around VMClarity**: VMClarity will only be successful if
+  we foster a community of users and developers around the project. The health
+  of VMClarity depends on organizing events, creating a forum for users to drive
+  requirements and use cases, and contributing to open-source projects related to
+  VMClarity. Building a strong community around VMClarity will help it be more
+  widely adopted and more effective at detecting and managing virtual machine
+  threats.
+
```

Comment:
Looking at the rendered version of the README I think we should flip Goal and Roadmap, so its Goals first and then Roadmap. It tells a better story this way, first you tell the high level goals, and then you say and this is how we intend to get there.